### ISSUE TITLE
Password text field

### ISSUE THREAD

**Issue by verma611** [TomSchimansky/CustomTkinter#647]
Can you make a password field using customtkiner, for example the instead of the password letters it displays bulletpoints to hide the password

**Comment by KeksTM** [comment]
Have you tried using Google?

For example: https://www.tutorialspoint.com/how-to-create-a-password-entry-field-using-tkinter

**Comment by verma611** [comment]
> Have you tried using Google?
> 
> For example: https://www.tutorialspoint.com/how-to-create-a-password-entry-field-using-tkinter

Yes... but there is no "show" argument for the text Feild. I have been using tkiner for a long time, but i recently heard about this
